scripts/build-triage-input.py

```
@@ -0,0 +1,340 @@
+        try:
+            patterns.append(json.loads(m.group(1)))
+        except json.JSONDecodeError:
+            sys.stderr.write(f"WARN: pattern issue body has invalid JSON; skipping\n")
+    return patterns
+
+
+def run_redact(text: str) -> str:
+    """Pipe through lib/redact-log.sh; fall back to identity if missing."""
+    if not REDACT_SH.exists():
+        return text
+    proc = subprocess.run(["bash", str(REDACT_SH)], input=text,
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)